validator/client/propose.go

```diff
@@ -240,9 +241,11 @@ func CreateSignedVoluntaryExit(
 		return nil, errors.Wrap(err, "gRPC call to get genesis time failed")
 	}
 	totalSecondsPassed := prysmTime.Now().Unix() - genesisResponse.GenesisTime.Seconds
-	currentEpoch := primitives.Epoch(uint64(totalSecondsPassed) / uint64(params.BeaconConfig().SlotsPerEpoch.Mul(params.BeaconConfig().SecondsPerSlot)))
+	if epoch == 0 {
```

Comment:
if a struct is used then there are no assumptions made and it should be overridden only if it's populated.